src/storybook/components/multiple-story-elements-wrapper/multiple-story-elements-wrapper.jsx

```diff
@@ -9,3 +9,5 @@ export const MultipleStoryElementsWrapper = ({ className, children }) => {
     </div>
   );
 };
+
+export default MultipleStoryElementsWrapper;
```

Comment:
same here, it's a hack which I think that we do not want to export right now

Reply:
Removed from `published-storybook-components`